src/cloudkit.cs

```diff
@@ -19,15 +19,155 @@ interface CKAsset : NSCoding, NSSecureCoding {
 		[Export ("fileURL", ArgumentSemantic.Copy)]
 		NSUrl FileUrl { get; }
 	}
+	
+	[iOS (10,0), Watch (3,0), TV (10,0), Mac (10,12, onlyOn64 : true)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface CKUserIdentityLookupInfo : NSSecureCoding, NSCopying
+	{
+		[Static, Internal, Export ("initWithEmailAddress:")]
+		IntPtr _FromEmail (string emailAddress);
+
+		[Static, Internal, Export ("initWithPhoneNumber:")]
+		IntPtr _FromPhoneNumber (string phoneNumber);
+
+		[Static, Internal, Export ("initWithUserRecordID:")]
+		CKUserIdentityLookupInfo FromRecord (CKRecordID userRecordID);
+
+		[Static]
+		[Export ("lookupInfosWithEmails:")]
+		CKUserIdentityLookupInfo[] GetLookupInfosWithEmails (string[] emails);
+
+		[Static]
+		[Export ("lookupInfosWithPhoneNumbers:")]
+		CKUserIdentityLookupInfo[] GetLookupInfosWithPhoneNumbers (string[] phoneNumbers);
+
+		[Static]
+		[Export ("lookupInfosWithRecordIDs:")]
+		CKUserIdentityLookupInfo[] GetLookupInfosWithRecordIds (CKRecordID[] recordIDs);
```

Comment:
or just `GetLookupInfos` ?!? right now this is a bit inconsistent with `DiscoverUserIdentity*` API